### ISSUE TITLE
False positive for play-akka-http-server_2.x-2.6.5.jar

### ISSUE THREAD

**Issue by albuch** [dependency-check/DependencyCheck#965]
False positive on library play-akka-http-server_2.11-2.6.5.jar - reported as `cpe:/a:akka:akka` and `cpe:/a:akka:http_server`
```xml
<dependency>
   <groupId>com.typesafe.play</groupId>
   <artifactId>play-akka-http-server_2.11</artifactId>
   <version>2.6.5</version>
</dependency>
```
We've used the following suppression rule. Please note the number placeholder for the scala version.
```
    <suppress>
        <notes><![CDATA[
   file name: play-akka-http-server_2.11-2.6.5.jar
   ]]></notes>
        <gav regex="true">^com\.typesafe\.play:play-akka-http-server_2\.\d+:.*$</gav>
        <cpe>cpe:/a:akka:akka</cpe>
        <cpe>cpe:/a:akka:http_server</cpe>
    </suppress>
```

**Comment by jeremylong** [comment]
Thanks!  I'll do a sprint cleaning up all the report FP before the next release.
